eslint.config.mjs

```diff
@@ -209,7 +213,7 @@ export default [
     }
   },
   {
-    files: ['_scripts/*.mjs'],
+    files: ['_scripts/**/*.mjs'],
```

Comment:
I think the one on `eslint-lint` & `eslint-lint-fix` should be updated too

Reply:
Good catch, thank you!

Reply:
Fixed!